### PR TITLE
data: DavidRaigozaJ

### PR DESCRIPTION
**Pull request by DavidRaigozaJ**
## Fixes Issue
This is the first commit of my LinkFree Repo.

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8096"></a>

### CODE REVIEW

File: DavidRaigozaJ [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            "icon": "FaGithub"
+        },
+        {
+            "name": "Visit our e-commerce",
+            "url": "https://pepeymatilda.co/",
+            "icon": "FaShoppingCart"
+        }
+    ]
+}
+
```
Comment (Vishrut19) [inline DavidRaigozaJ:32]: This extra space is not needed you can remove it.

**Review by Vishrut19 — CHANGES_REQUESTED** [review]

**Review by Sindhuinti — APPROVED** [review]
Looks good to me 👍🏻

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please move your file to the `data` folder and add `.json` at the end. 

Your file should be: `data/DavidRaigozaJ.json`

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: [https://linkfree.io/docs/quickstart to help you update this Pull Request.](https://linkfree.io/docs/quickstart-json)

**Comment by SaraJaoude** [comment]
> Looks good to me 👍🏻

@Sindhuinti please do not approve a Pull Request without doing a proper check as this can be confusing for the author.  As you can see this Pull Request is not ready to be merged.

**Review by SaraJaoude — APPROVED** [review]

**Comment by DavidRaigozaJ** [comment]
Thank you!

**Pull request merged**